internal/test/report.go

```diff
@@ -7,17 +7,17 @@ import (
 
 // Report prints the results of the testing to the IO writer.
 func Report(w io.Writer, rs Results) error {
-	_, err := fmt.Fprintf(w, "%d/%d tests passed", rs.CountPassed(), rs.Count())
+	_, err := fmt.Fprintf(w, "%d/%d tests passed\n", rs.CountPassed(), rs.Count())
```

Comment:
We don't need both this log and both the logs below. I think just remove this one.